config.docker.js

```diff
@@ -15,6 +15,24 @@ const CONFIG = {
   ssl: process.env.FLOOD_ENABLE_SSL === 'true' || process.env.FLOOD_ENABLE_SSL === true,
   sslKey: '/data/flood_ssl.key',
   sslCert: '/data/flood_ssl.cert',
+  diskUsageService: {
+    dontWatchMountPoints: [
```

Comment:
I think you can't just enumerate files our mountpoints here because they can be different for each distro. We need to find a cross-distro way to do that.

Reply:
Those are paths in the container, I don't know if the host distro affects this. I only tried on ubuntu.

Reply:
My bad, I missed it was for docker only.